exporter/signalfxexporter/internal/dimensions/dimclient.go

```diff
@@ -325,3 +325,8 @@ func (dc *DimensionClient) makePatchRequest(dim *DimensionUpdate) (*http.Request
 
 	return req, nil
 }
+
+func (dc *DimensionClient) Stop() {
+	dc.requestSender.Stop()
```

Comment:
I think we should gracefully wait for the request to finish. The context-based behavior ensures that

Reply:
ok